pkg/common/common.go

```diff
@@ -8,13 +8,11 @@ import (
 const ReadUser = "read"
 
 func IsLoggingEnabled(lc loggedcluster.Interface) bool {
-
 	// Logging should be enabled when all conditions are met:
 	//   - logging label is set and true on the cluster
 	//   - cluster is not being deleted
-	//   - TODO(theo) global logging flag is enabled
-
-	return lc.GetLoggingLabel() == "true" && lc.GetDeletionTimestamp().IsZero() && lc.GetEnableLoggingFlag()
+	//   - global logging flag is enabled
+	return lc.IsLoggingEnabled() && lc.GetDeletionTimestamp().IsZero() && lc.GetEnableLoggingFlag()
```

Comment:
We had named the lc function `GetLoggingLabel` because it is not enough of a condition for logging to be enabled.
As seen here, `common.IsLoggingEnabled` depends on multiple conditions, and having the logging label is one of them.

I thing both function namings are confusing, but I obviously prefer `GetLoggingLabel` because that's what I chose when I wrote it, after hesitating between those 2 names :grin:

Reply:
Well, I would agree but if you look at the function code, it now does more than just getting a label value right? Ir actually checks that logging is enabled for the logged cluster

Reply:
We could rename it to hasLoggingEnabled if you prefer?

Reply:
ok for `hasLoggingEnabled`, at least that's a different function name.

Reply:
Done :)